apps/etterlatte-saksbehandling-ui/client/src/components/person/hendelser/HendelseBeskrivelse.tsx

```diff
@@ -317,13 +314,23 @@ export const HendelseBeskrivelse = (props: { hendelse: Grunnlagsendringshendelse
     case 'GRUNNBELOEP':
       return (
         <Header>
-          <BodySmall>Hendelse fra {grunnlagsendringsKilde(hendelse.samsvarMellomKildeOgGrunnlag.type)}</BodySmall>
-          <BodySmall>{grunnlagsendringsBeskrivelse[hendelse.samsvarMellomKildeOgGrunnlag.type]}</BodySmall>
+          <KortTekst size="small">
+            Hendelse fra {grunnlagsendringsKilde(hendelse.samsvarMellomKildeOgGrunnlag.type)}
+          </KortTekst>
+          <KortTekst size="small">{grunnlagsendringsBeskrivelse[hendelse.samsvarMellomKildeOgGrunnlag.type]}</KortTekst>
         </Header>
       )
   }
 }
 
+const Broedtekst = ({ children }: { children: ReactNode | Array<ReactNode> }) => {
+  return (
+    <KortTekst size="small" weight="semibold">
+      {children}
+    </KortTekst>
+  )
+}
+
```

Comment:
Skulle kanskje heller sett at du tok i bruk mer DS react enn å innføre enda et komponent 😅 

Ser at mange av disse komponentene har en wrapper. I mitt hode burde kanskje heller wrapperen ha ansvar for bredden, og en `<Label>` ha ansvar for å gjøre teksten fet.